action-scripts/merge_pr_jsons.py

```diff
@@ -125,11 +138,28 @@ def main():
             result["transactions"] = []
             for file in fs:
                 result["transactions"] += file["transactions"]
+                #  Check for gauge adds and generate checkpoint list
+
+                for tx in file["transactions"]:
+                    if tx["contractMethod"]["name"] == "addGauge":
+                        try:
+                            gauge_chain = tx["contractInputsValues"]["gaugeType"]
+                            if gauge_chain != "Ethereum":
+                                if gauge_chain not in gauge_lists_by_chain.keys():
```

Comment:
You can simplify by doing this:
```python
from collections import defaultdict

gauge_lists_by_chain = defaultdict(list)
gauge_lists_by_chain[gauge_chain].append(tx["contractInputsValues"]["gauge"])
```

It will initialize empty lists for any new values so you don't have to do `if gauge_chain not in gauge_lists_by_chain.keys():` this check